libs/service-portal/core/src/utils/amountFormat.ts

```diff
@@ -1,4 +1,5 @@
 export const amountFormat = (value: number): string => {
+  if (value === 0) return '0 kr.'
```

Comment:
Is this line unnecessary perhaps?
The previous version would return 0 kr. still if the value is `0` no?